src/reactionsystem.jl

```diff
@@ -1479,4 +1479,4 @@ function validate(rs::ReactionSystem, info::String = "")
 end
 
 # Checks if a unit consist of exponents with base 1 (and is this unitless).
-unitless_exp(u) = istree(u) && (operation(u) == ^) && (arguments(u)[1] == 1)
+unitless_exp(u) = iscall(u) && (operation(u) == ^) && (arguments(u)[1] == 1)
```

Comment:
Not sure why this one and others was missed in the previous on. Literally did a ctrl+f for `istree`, but I guess something was up